docker-compose.yml

```
@@ -76,3 +88,14 @@ services:
       - .:/api
     depends_on:
       - lms
+
+  nginx:
+    image: nginx:1.13
+    ports:
+      - "8073:8072"
```

I think the following will be less confusing.
```suggestion
      - "8080:8080"
```